.github/workflows/ubuntu.yml

```diff
@@ -13,11 +13,17 @@ jobs:
         .github/workflows/dependencies/gcc.sh
     - name: build WarpX
       run: |
-        cmake -S . -B build            \
+        cmake -S . -B build_3D         \
           -DCMAKE_VERBOSE_MAKEFILE=ON  \
           -DWarpX_MPI=OFF              \
           -DWarpX_QED=OFF
-        cmake --build build -j 2
+        cmake --build build_3D -j 2
+        cmake -S . -B build_RZ         \
+          -DCMAKE_VERBOSE_MAKEFILE=ON  \
+          -DWarpX_DIMS=RZ              \
+          -DWarpX_MPI=OFF              \
+          -DWarpX_QED=OFF
```

Comment:
Just checking: the default values of the other flags not listed here is `OFF`?

Reply:
PSATD is off by default, yes: https://warpx.readthedocs.io/en/latest/install/cmake.html#build-options
And this entry does not install FFTW, so it definitely crashes (as the first commit shows :) )